engine/graphics/src/opengl/graphics_opengl.cpp

```diff
@@ -1894,7 +1903,7 @@ static uintptr_t GetExtProcAddress(const char* name, const char* extension_name,
             glBindRenderbuffer(GL_RENDERBUFFER, rt->m_DepthStencilBuffer);
 #ifdef GL_DEPTH_STENCIL_ATTACHMENT
             // if we have the capability of GL_DEPTH_STENCIL_ATTACHMENT, create a single combined depth-stencil buffer
-            glRenderbufferStorage(GL_RENDERBUFFER, GL_DEPTH_STENCIL, rt->m_BufferTextureParams[param_buffer_index].m_Width, rt->m_BufferTextureParams[param_buffer_index].m_Height);
+            glRenderbufferStorage(GL_RENDERBUFFER, DMGRAPHICS_RENDER_BUFFER_FORMAT_DEPTH_STENCIL, rt->m_BufferTextureParams[param_buffer_index].m_Width, rt->m_BufferTextureParams[param_buffer_index].m_Height);
```

Comment:
Depending on platform, it's defined as either `GL_DEPTH_STENCIL` or `GL_DEPTH24_STENCIL8_OES` (if `GL_DEPTH_STENCIL_OES` is defined)